### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gcampax**

### CODE REVIEW

**Comment by gcampax** [comment]
@jgd5 please take a look to the new API in dbproxy, i'm not sure `/by-:field` is valid

File: go/dbproxy/local_table.go [hunk 1]
```diff
@@ -65,6 +65,44 @@ func localTableGetOne(c *gin.Context) {
...
+		return
+	}
+
+	field := c.Param("field")
+	if !contains(m.Fields(), field) {
+		c.JSON(http.StatusNotFound, gin.H{"error": "invalid field"})
+		return
+	}
+
+	value := c.Param("value")
```
Comment (jgd5) [inline go/dbproxy/local_table.go:97]: Can `value` be empty or we should check `len(value) > 0`?

**Review by jgd5 — APPROVED** [review]
LGTM
verified url pattern:
`"/localtable/:name/by-:field/:userid/:value"`

File: go/dbproxy/local_table.go (hunk 1 above)
Comment (gcampax) [reply go/dbproxy/local_table.go:97]: Would the URL pattern match at all if the value is empty?

Comment (jgd5) [reply go/dbproxy/local_table.go:97]: The URL pattern will not match since value the last component. 
So the code looks good.

**Pull request merged**